test/functionalTest/cases/000_ngsi10_convenience/attribute_value_instance.test

```diff
@@ -160,12 +160,15 @@ Date: REGEX(.*)
 </contextAttributeResponse>
 2: ++++++++++++++++++++
 HTTP/1.1 200 OK
-Content-Length: 685
+Content-Length: 755
 Content-Type: application/xml
 Date: REGEX(.*)
 
 <?xml version="1.0"?>
 <appendContextElementResponse>
+  <entityId type="" isPattern="false">
```

Comment:
I think is again another side-effect of have now "/attribute" exactly equal to the other operation.

NTC